### ISSUE TITLE
[BUG] EMFILE: too many open files, watch node_modules/youtube-dl/bin

### ISSUE THREAD

**Issue by chjohn22** [Tzahi12345/YoutubeDL-Material#456]
I exported the container settings to .json, deleted the old container and recreated the new one with the modified settings of the "stable" container. It tries to update YoutubeDL-material but fails, then crashes.

Logs: 
`﻿2021-09-19T09:16:20.208Z INFO: Found new update for youtube-dl. Updating binary...`

`2021-09-19T09:16:20.213Z ERROR: Failed to update youtube-dl - Error: EMFILE: too many open files, watch node_modules/youtube-dl/bin`
`/app/app.js:840 `
             `  watcher.close(); `
                   `   ^ `
`TypeError: Cannot read property 'close' of undefined 
at /app/app.js:840:25`
`[90m    at FSReqCallback.oncomplete (fs.js:156:23)[39m `

**Environment**
- Docker tag: nightly

**Comment by Tzahi12345** [comment]
Strange, looks like the container ran out of allowable number of files to watch. Seems like it's something to configure on the host, I found this comment that may help:

https://github.com/cdr/code-server/issues/628#issuecomment-636526989

Let me know if that helps. If not, perhaps we have a deeper issue here. I did put in [a fix](https://github.com/Tzahi12345/YoutubeDL-Material/commit/33f23c3ca9a04d9e3e5467fb4c664c5e210dcfab) to avoid the unhelpful error but it doesn't look like it'll solve the problem on its own.
